Sources/Customization/Protocols/OPTEventHandler.swift

```diff
@@ -0,0 +1,29 @@
+/****************************************************************************
+* Copyright 2019, Optimizely, Inc. and contributors                        *
+*                                                                          *
+* Licensed under the Apache License, Version 2.0 (the "License");          *
+* you may not use this file except in compliance with the License.         *
+* You may obtain a copy of the License at                                  *
+*                                                                          *
+*    http://www.apache.org/licenses/LICENSE-2.0                            *
+*                                                                          *
+* Unless required by applicable law or agreed to in writing, software      *
+* distributed under the License is distributed on an "AS IS" BASIS,        *
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+* See the License for the specific language governing permissions and      *
+* limitations under the License.                                           *
+***************************************************************************/
+    
+import Foundation
+
+public protocol OPTEventHandler {
```

Comment:
I'd like to discuss.  We want to deprecate DefaultEventDispatcher and explain that dispatch now just sends and is part of event dispatcher.  Then, we create a new EventDispatcher that is HttpEventProcessor.